spec/controllers/queue_items_controller_spec.rb

```diff
@@ -105,6 +172,24 @@
         expect(response).to redirect_to login_path 
       end
     end
+
+    describe "POST update" do
+      it "should redirect to login page" do
```

Comment:
The word `should` should be deleted not added. Its not needed - read more about this http://betterspecs.org/#should